doc/content/getting-started/migrating/migrating-from-ce-to-ch/establish-new-session/_index.md

```diff
@@ -0,0 +1,114 @@
+
+<!--more-->
+
+Read the instructions below for your specific device.
+
+## OTAA
+
+Migrating an OTAA device without persisting its active session means the device will establish a new session with {{% tts %}} Cloud, i.e. it will have to perform a new join on {{% tts %}} Cloud network after migration. The device will negotiate about network parameters with {{% tts %}} Cloud Network Server, and during that negotiation, the device will be assigned with a new DevAddr from {{% tts %}} Cloud DevAddr block.
+
+Since {{% tts %}} Community Edition and {{% tts %}} Cloud are both connected to Packet Broker, Packet Broker will be able to route your device's traffic to {{% tts %}} Cloud even if your gateway stays connected to {{% tts %}} Community Edition, i.e. you don't have to migrate your gateway to {{% tts %}} Cloud, only your device. However, if you want to migrate your gateway to {{% tts %}} Cloud too, see [Migrating Gateways]({{< ref "/getting-started/migrating/gateway-migration" >}}) for instructions.
+
+To migrate your devices from {{% tts %}} Community Edition to {{% tts %}} Cloud without active sessions, follow the steps described below.
+
+{{< tabs/container "Console" "CLI">}}
+
+{{< tabs/tab "Console" >}}
+
+Using the Console is covenient only when you have a few devices to migrate. For larger groups of devices, we recommend using the CLI or the migration tool.
+
+Recreate your device on {{% tts %}} Cloud. See [Adding Devices]({{< ref "/devices/adding-devices" >}}) for instructions on creating a device. You can reuse the DevEUI, AppEUI/JoinEUI and AppKey values from {{% tts %}} Community Edition. You can also generate new values on {{% tts %}} Cloud, but then you will need to re-program your OTAA device using those values.
+
+{{< /tabs/tab >}}
+
+{{< tabs/tab "CLI" >}}
+
+First, configure your CLI to connect to {{% tts %}} Community Edition. See [Configuring the CLI]({{< ref "/getting-started/cli/configuring-cli" >}}) guide for instructions. Make sure you also perform a [Login with the CLI]({{< ref "/getting-started/cli/login" >}}) to {{% tts %}} Community Edition.
+
+{{< note >}} We recommend to use the latest version of the CLI. Instructions for upgrading the CLI if you already have it installed are available in the [Installing the CLI]({{< ref "/getting-started/cli/login" >}}) guide. {{</ note >}}
+
+Now, use the CLI to export your device's description from {{% tts %}} Community Edition:
+
+```bash
+ttn-lw-cli end-devices get --application-id <app-id> --device-id <device-id> --name --description --lorawan-version --lorawan-phy-version --frequency-plan-id --supports-join --root-keys --mac-settings > device-description.json
+```
+
+The command above will export your device's description to the `device-description.json` file in the current folder. Open the file with a text editor and remove the following fields: `join_server_address`, `network_server_address` and `application_server_address`.
```

Comment:
Hmm this is very inconvenient.

Is this even necessary?

Why don't we have support for TTS to TTS migration in `ttn-lw-migrate`?

Reply:
I guess that's just a temporary workaround, it was described in #542. We should get that support for migration tool with https://github.com/TheThingsNetwork/lorawan-stack-migrate/issues/42